pkg/cmd/convert.go

```diff
@@ -62,15 +69,39 @@ func GetValidatedObject(v *viper.Viper, cmd *cobra.Command, out, outerr io.Write
 	}
 
 	var ocObjects []*object.OpenCompose
+	var data []byte
+	var err error
 	for _, file := range files {
-		data, err := ioutil.ReadFile(file)
-		if err != nil {
-			return nil, fmt.Errorf("unable to read file '%s': %s", file, err)
+		// Check if the passed resource is a URL or not
+		if strings.HasPrefix(file, "http://") || strings.HasPrefix(file, "https://") {
+
```

Comment:
we can leave out this empty line after `if` so it looks like the rest of the code